Doc/library/mmap.rst

```diff
@@ -342,3 +342,21 @@ MADV_* Constants
    Availability: Systems with the madvise() system call.
 
    .. versionadded:: 3.8
+
+.. _map-constants:
+
+MAP_* Constants
++++++++++++++++
+
+.. data:: MAP_SHARED
+          MAP_PRIVATE
+          MAP_DENYWRITE
+          MAP_EXECUTABLE
+          MAP_ANON
+          MAP_ANONYMOUS
+          MAP_POPULATE
+
+    These are the various flags that can be passed to :meth:`mmap.mmap`. Note that some options might not be present on some systems.
+
+    .. versionchanged:: 3.10
+      Add MAP_POPULATE
```

Comment:
Is it correctly rendered? Maybe fix the indentation. I also suggest to add a final "." to your sentence :-)

Reply:
Oops. I just fixed this (and changed to past tense to better match other change notices).